<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeForm</class>
 <widget class="QWidget" name="homeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>50</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>460</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>温度</string>
         </property>
         <widget class="Line" name="line">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="tempNumber">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QSpinBox" name="tempspinBox">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>100</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="value">
           <number>35</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>100</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>阈值：</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>40</y>
            <width>51</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;℃&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="tempButton">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>100</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>设备状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>设备类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>阈值设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="tempStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>35</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_30">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>检测</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <widget class="Line" name="line_6">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="nonepixlabel">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>30</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_39">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>65</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="noneStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>30</y>
            <width>31</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>27</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>水位</string>
         </property>
         <widget class="Line" name="line_2">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QSpinBox" name="watlspinBox">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>100</y>
            <width>42</width>
            <height>22</height>
           </rect>
          </property>
          <property name="value">
           <number>70</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>100</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>阈值：</string>
          </property>
         </widget>
         <widget class="QPushButton" name="watlButton">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>100</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_31">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>检测</string>
          </property>
         </widget>
         <widget class="QLabel" name="watlStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>35</y>
            <width>35</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>设备状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>设备类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>阈值设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QProgressBar" name="watlprogressBar">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>141</width>
            <height>31</height>
           </rect>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>水泵</string>
         </property>
         <widget class="Line" name="line_4">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_35">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>控制</string>
          </property>
         </widget>
         <widget class="QLabel" name="pumpStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>35</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>设备状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>设备类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>阈值设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="pumpvaluelabel">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="pumppixlabel">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>40</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>浊度</string>
         </property>
         <widget class="Line" name="line_5">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="turbpixlabel">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>40</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_37">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>检测</string>
          </property>
         </widget>
         <widget class="QLabel" name="turbStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>35</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>设备状态</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>设备类型</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>阈值设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="turbvaluelabel">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>热灯</string>
         </property>
         <widget class="Line" name="line_3">
          <property name="geometry">
           <rect>
            <x>220</x>
            <y>20</y>
            <width>20</width>
            <height>101</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_33">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lampStatelabel">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>35</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>离线</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>240</x>
            <y>30</y>
            <width>62</width>
            <height>91</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>设备状态：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>设备类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>阈值设置</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="label_36">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>70</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>控制</string>
          </property>
         </widget>
         <widget class="QLabel" name="lampvaluelabel">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>70</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="lamppixlabel">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>40</y>
            <width>81</width>
            <height>81</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
